packages/contracts/tests/js/ZEROIssuanceArithmeticTest.js

```diff
@@ -34,6 +37,11 @@ const repeatedlyIssueZERO = async (stabilityPool, timeBetweenIssuances, duration
   }
 };
 
+const calculateSOVAccrual = (initialLastIssuanceTime, latestLastIssuanceTime, totalZUSDDeposits) => {
```

Comment:
you updated the smart contract function to do all multiplications before divisions but forgot to update this here accordingly.

Reply:
while this function should be correct, it would still be beneficial to also have a test case which is checking against a certain expected value which is calculated beforehand. e.g. with a  total ZUSD deposit of 100k and 5% apr after half a year, we would expect 2.5k in SOV issued.

Reply:
also, please add a test where the APR changes and make sure that the total amount of SOV is issued.
for example: 5% for half a year and then 10% for half a year on a total 100k ZUSD deposit -> 7.5k SOV should be issued after one year

Reply:
Added all of the test cases